sycl/test/esimd/lsc_block_load_store.cpp

```diff
@@ -0,0 +1,59 @@
+// RUN: not %clangxx -fsycl -fsyntax-only -Wno-unused-command-line-argument %s 2>&1 | FileCheck %s --implicit-check-not="warning:" --implicit-check-not="error:"
+
+// This test checks that lsc_block_load/store API gets successfully compiled.
+
+#include <limits>
+#include <sycl/ext/intel/esimd.hpp>
+#include <sycl/sycl.hpp>
+#include <utility>
+
+using namespace sycl::ext::intel::esimd;
+using namespace sycl::ext::intel::experimental::esimd;
+using namespace sycl;
+
+SYCL_EXTERNAL void
+kernel1(accessor<int, 1, access::mode::read_write, access::target::device> &buf)
+    SYCL_ESIMD_FUNCTION {
+  simd<int, 32> v1(0, 1);
+  auto v0 = lsc_block_load<int, 32>(buf, 0);
+  v0 = v0 + v1;
+  lsc_block_store<int, 32>(buf, 0, v0);
+}
+
+SYCL_EXTERNAL void kernel2(int *ptr) SYCL_ESIMD_FUNCTION {
```

Comment:
Is this test related to the accessor checks?
It may be ok to keep it, but seems unrelated to the topic of this PR.